packages/directlinespeech/src/playCognitiveServicesStream.js

```diff
@@ -0,0 +1,122 @@
+/* eslint no-magic-numbers: ["error", { "ignore": [8, 16, 32, 128, 1000, 32768, 2147483648] }] */
```

Comment:
I understand this came from the Speech team. If that's true let's add a comment denoting as such, and maybe provide a point of contact.